src/integrator/LangevinThermostatOnGroup.py

```diff
@@ -0,0 +1,52 @@
+#  Copyright (C) 2016
+#      Jakub Krajniak (jkrajniak at gmail.com)
+#
+#  This file is part of ESPResSo++.
+#
+#  ESPResSo++ is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  ESPResSo++ is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+r"""
+***************************************************
+**espressopp.integrator.LangevinThermostatOnGroup**
+***************************************************
+
+Thermalize particles in the ParticleGroup only.
+
```

Comment:
Could we also include a few lines of example python script here? Showing creation of particle group and then adding thermostat extension to integrator.